src/rules/if_in_if.ts

```diff
@@ -17,7 +17,7 @@ export class IfInIf extends ABAPRule {
   }
 
   public getDescription(): string {
-    return "IF in IF";
+    return "IF in IF. Use IF cond1 AND cond2 instead. ";
```

Comment:
there are more cases, but anyhow, this description is better

Reply:
keeping it as is then